app/serializers/page_serializer/layings_show_page_serializer.rb

```diff
@@ -0,0 +1,64 @@
+module PageSerializer
+  class LayingsShowPageSerializer < LaidThingShowPageSerializer
+    # Initialise a Statutory InstrumentLaying
+    # @param [ActionDispatch::Request] request the current request object.
+    # @param [<Grom::Node>] laying a Grom::Node object of type Laying.
+    # @param [Array<Hash>] data_alternates array containing the href and type of the alternative data urls.
+    def initialize(request: nil, laying:, data_alternates: nil)
+      @laying = laying
+
+      super(request: request, laid_thing: @laying.laid_thing, data_alternates: data_alternates)
+    end
+
+    private
+
+    def meta
+      super(title: title)
+    end
+
+    def title
+      @laying.procedure_step.try(:procedureStepName)
```

Comment:
They both work as fall backs, but have different content. I think it might be worth making the title default a separate ticket, but we need to have a discussion about what the correct fall back is. Allan's is different to the ones we used.

Reply:
Ok, making it a separate ticket is fine and we can make the decision later.